### DISCUSSION TITLE
How to use MacOS (Apple) emoji in wezterm.

### DISCUSSION THREAD

**Discussion by BlueFalconHD** [wezterm/wezterm#2354] [Q&A]
How are you supposed to use the MacOS emojis that look good versus the weird looking emojis in wezterm by default. Is this something that can be achieved by configuration, or just unchangeable.

**Comment by BlueFalconHD** [comment]
| Apple | Wezterm | 
|-------|---------|
|![image](https://user-images.githubusercontent.com/72631767/182678575-224eec84-56ab-462c-8e09-ba84c6a4a022.png)|![image](https://user-images.githubusercontent.com/72631767/182678386-be917ad6-fdfb-4f9f-a08a-b6bd19043384.png)|

**Reply by wez** [reply]
Answered in your duplicate issue: https://github.com/wez/wezterm/issues/2353
